stixcore/products/level0/scienceL0.py

```diff
@@ -510,13 +523,17 @@ def from_levelb(cls, levelb, parent=''):
                     'counts', 'counts_comp_err']
         data['control_index'] = np.ubyte(0)
 
-        return cls(service_type=packets.service_type,
+        prod = cls(service_type=packets.service_type,
                    service_subtype=packets.service_subtype,
                    ssid=packets.ssid,
                    control=control,
                    data=data,
                    idb_versions=idb_versions,
-                   packets=packets)
+                   packets=packets,
+                   history=header_history)
+
+        prod.add_additional_header_keywords(additional_header_keywords)
+        return prod
```

Comment:
@samaloney here a list of keywords is added
there is also a  prod.add_additional_header_keyword() if you would like to add a single one
but most likely as the product is created as last step you have to store the keyword in a list during processing